lib/diagnostic.js

```diff
@@ -6,20 +6,33 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function () {
-
+const normalizeWords = function normalizeWords(string) {
+  let wordList = string.splits(/\s+/);
```

Comment:
this was correct except for the a small typo where you called ```string.splits``` instead of ```string.split``` which sadly broke everything